PendingReleaseNotes

```diff
@@ -21,6 +21,8 @@
 * RGW: Adding missing quotes to the ETag values returned by S3 CopyPart,
   PostObject and CompleteMultipartUpload responses.
 * RGW: Added support for S3 GetObjectAttributes.
+* RGW: Facilitate running multiple RGWs processes on same host by leveraging the
+  so_reuseport socket option.
```

Comment:
release note should mention the `so_reuseport` frontend option now that it's not the default:
> Added beast frontend option `so_reuseport` to allow RGW processes on the same host to share the same port number.

Reply:
done